sdks/go/pkg/beam/core/util/bundle/bundle.go

```diff
@@ -0,0 +1,36 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle
+
+import "context"
+
+type key string
+
+const bundleKey key = "beam:bundle"
+
+// SetID sets the id of the current Bundle.
+func SetID(ctx context.Context, id string) context.Context {
```

Comment:
These tiny helpers seem to better belong in the metrics package instead of a separate one. Ditto for ptransform.

Reply:
Done